spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Tracer.java

```diff
@@ -104,7 +122,15 @@
 	 */
 	Span close(Span span);
 
+	/**
+	 * Returns a wrapped {@link Callable} that is prepared for span data passing
```

Comment:
hmm not sure what this means. maybe be more concrete? Returns a callable which will be recorded as a span in the current trace?